generators/Exercises/RationalNumbers.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Generators.Input;
+using Generators.Output;
+using Newtonsoft.Json.Linq;
+
+namespace Generators.Exercises
+{
+    public class RationalNumbers : GeneratorExercise
+    {
+        protected override string RenderTestMethodBodyAssert(TestMethodBody testMethodBody)
+        {
+            if (testMethodBody.CanonicalDataCase.Expected.GetType().Name.Equals("Double"))
+			     return RenderRationalNumberAssert(testMethodBody);
+            return base.RenderTestMethodBodyAssert(testMethodBody);
+        }
+
+        private static string RenderRationalNumberAssert(TestMethodBody testMethodBody)
+        {		
+            const string template = "Assert.Equal({{ ExpectedParameter }}, {{ TestedValue }}, precision: [precision]);";
+            return TemplateRenderer.RenderInline(template.Replace("[precision]", precision(testMethodBody.CanonicalDataCase.Properties["expected"]).ToString()),
+			                 testMethodBody.AssertTemplateParameters);
+        }
+				
+	private static int precision(object rawValue) => 
```

Comment:
Could you use PascalCasing here? To better adhere to the default [naming conventions](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members).

Reply:
OK, I'll do the fix. (I have used to use naturally for quite a few years the previous version of VS naming conventions where private and protected members are "camelCased" and constants - "UPPER_CASED_WITH_UNDERSCORES_SEPARATING_CONSTANT_NAMES_WORDS_PARTS".)